docs/products/kafka/howto/schema-registry.rst

```diff
@@ -0,0 +1,149 @@
+``BOOTSTRAPSERVERS``               Service URI for Kafka connection, can be obtained from the `Aiven console <https://console.aiven.io/>`_, service Overview tab
+``KEYSTORE``                       Path to the keystore
+``KEYSTOREPASSWORD``               Password for the keystore
+``TRUSTSTORE``                     Path to the truststore
+``TRUSTSTOREPASSWORD``             Password for the truststore
+``SSLKEYPASSWORD``                 The password of the private key in the key store file
+``SCHEMAREGISTRYURL``              Service Registry URI for Apache Kafka connection
+``SCHEMAREGISTRYUSER``             Service Registry username, can be obtained from the `Aiven console <https://console.aiven.io/>`_, service Overview tab
+``SCHEMAREGISTRYPASSWORD``         Service Registry password, can be obtained from the `Aiven console <https://console.aiven.io/>`_, service Overview tab
+``TOPIC_NAME``                     Apache Kafka topic name to use
+=============================      ===========================================================================================================================
+
+
+Create version 1 of the Avro schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create an Avro schema, you need a definition file. As example you can use a **click record** schema defined in JSON and stored in a file named ``ClickRecord.avsc`` containing the following:
+
+::
+
+    {"type": "record",
+      "name": "ClickRecord",
+      "namespace": "io.aiven.avro.example",
+      "fields": [
+         {"name": "session_id", "type": "string"},
+         {"name": "browser", "type": ["string", "null"]},
+         {"name": "campaign", "type": ["string", "null"]},
+         {"name": "channel", "type": "string"},
+         {"name": "referrer", "type": ["string", "null"], "default": "None"},
+         {"name": "ip", "type": ["string", "null"]}
+      ]
+    }
+
+The JSON configuration above defines a schema named ``ClickRecord`` in the namespace ``io.aiven.avro.example`` with fields ``session_id``, ``browser``, ``campaign``, ``channel``, ``referrer`` and ``ip`` and related data types. 
+
+Once the schema is defined, you need to compile it, and it can be done **manually** or **automatically**.
+
+Manual schema compilation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case of manual schema compilation, download ``avro-tools-1.11.0.jar`` from https://avro.apache.org/releases.html#Download or via maven using the following::
+
+    mvn org.apache.maven.plugins:maven-dependency-plugin:2.8:get -Dartifact=org.apache.avro:avro-tools:1.11.0:jar -Ddest=avro-tools-1.11.0.jar
+
+The schema defined in the previous step, can be now compiled to produce a Java class ``ClickRecord.java`` in the ``io.aiven.avro.example`` package (taken from the ``namespace`` parameter)::
+
+    java -jar avro-tools-1.11.0.jar compile schema ClickRecord.avsc .
+
```

Comment:
I think would be useful to include expected output over there (fe `*.class` files)

Reply:
the issue is that in fact it generates lots of code for that (about 1k lines....)
I will try to see if it is possible to add here small but significant code-snippet from the whole result of generation

Reply:
Oh, I mean something like `When the generation finishes successfully, the `ClickRecord.java` should appear <somewhere>`, just to make expectations explicit.

Reply:
> a Java class ``ClickRecord.java`` in the ``io.aiven.avro.example`` package
I guess it is mentioned here.
Or probably makes sense to mentioned that the package structure also will be generated in case of absence

Reply:
done